docs/core/classes.md

```diff
@@ -185,6 +185,92 @@ aware that programs where private members are accessed from outside of their own
 class will work in the source version, but will fail to work in the build
 version!_
 
+### Cached Objects
```

Comment:
Isn't this a change in terminology?  Currently we refer to this as [Universal] Object ID, eg https://qooxdoo.org/documentation/v7.6/#/core/object_id

I've just seen that page is not linked to in the navigation, please can you add that too

Reply:
Please remember that private variables are mangled in source build too,